### PR TITLE
[joplin-server] new chart

### PR DESCRIPTION
**Pull request by auricom**
[Joplin-server](url) chart based on the dependent 'common' chart.
Used to sync multiple joplin clients.

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] (optional) Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [x] (optional) Variables are documented in the README.md

### CODE REVIEW

**Comment by bjw-s** [comment]
It looks like you are missing the README helm-doc template files that have been added recently. Please make sure you add these as well. That way the README should be automatically generated. 🙂

**Comment by auricom** [comment]
Ok, I think I understand. Nice feature !

**Comment by angelnu** [comment]
@auricom - if you enable the workflow action in your fork it will check that the helm doc is also updated in the same way as we do it in the main repository.

**Comment by auricom** [comment]
Thanks for helping me with this. I added the generated README.md

File: charts/joplin-server/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - https://github.com/laurent22/joplin/tree/dev/packages/server
+  - https://github.com/laurent22/joplin
+  - https://hub.docker.com/r/joplin/server
+maintainers:
+  - name: auricom
+    email: k8s-at-home@xpander.eml.cc
+dependencies:
+  - name: common
+    repository: https://k8s-at-home.com/charts/
+    version: 2.2.0
```
Comment (angelnu) [inline charts/joplin-server/Chart.yaml:21]: could you use the latest (3.1.0)?

File: charts/joplin-server/values.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 22300
+
+env:
+  # APP_BASE_URL: https://joplin.domain
```
Comment (angelnu) [inline charts/joplin-server/values.yaml:16]: Could you annotate this values? 
https://github.com/norwoodj/helm-docs

File: charts/joplin-server/Chart.yaml (hunk 1 above)
Comment (angelnu) [reply charts/joplin-server/Chart.yaml:21]: In the meanwhile the common has moved and should look like:
```
- name: common
  repository: https://library-charts.k8s-at-home.com
  version: 1.0.0
```

**Comment by onedr0p** [comment]
@all-contributors please add @auricom for code

File: charts/joplin-server/values.yaml (hunk 2 above)
Comment (auricom) [reply charts/joplin-server/values.yaml]: Ok, I added annotation like you did on [librespeed chart](https://github.com/angelnu/charts/blob/master/charts/librespeed/values.yaml)

**Pull request closed without merging**